src/tests/Interop/COM/NativeServer/DispatchCoerceTesting.h

```diff
@@ -0,0 +1,288 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include "Servers.h"
+
+/*template<typename T>
+HRESULT VerifyValues(_In_ const T expected, _In_ const T actual)
+{
+    return (expected == actual) ? S_OK : E_INVALIDARG;
+}
+
+VARIANTARG *NextArg(_In_ VARIANTARG *args, _Inout_ size_t &currIndex)
+{
+    return (args + (currIndex--));
+}*/
```

Comment:
Is this needed?

Reply:
They are commented out and I'm unclear what `VerifyValues` is being used below.

Reply:
Because all the .h of servers are included together into a single cpp file. Not sure about the best practice to avoid duplicated definition here.

Reply:
Okay, then the comments mean nothing and should be removed.